### PR TITLE
Add warning about low public key exponent

### PR DESCRIPTION
**Pull request by Amxx**
Fixes #???? 

#### PR Checklist

- [ ] Tests
- [ ] Documentation
- [ ] Changeset entry (run `npx changeset add`)

### CODE REVIEW

File: contracts/utils/cryptography/RSA.sol [hunk 1]
```diff
@@ -36,8 +36,12 @@ library RSA {
      * 2048 bits. If you use a smaller key, consider replacing it with a larger, more secure, one.
      *
      * WARNING: This verification algorithm doesn't prevent replayability. If called multiple times with the same
-     * digest, public key and (valid signature), it will return true every time. Consider including an onchain nonce or
-     * unique identifier in the message to prevent replay attacks.
+     * digest, public key and (valid signature), it will return true every time. Consider including an onchain nonce
+     * or unique identifier in the message to prevent replay attacks.
+     *
+     * WARNING: This verification algorithm support any exponent. NIST recommendation are to use `65537` (or higher).
+     * That is the default value used by many libraries, such as openssl. Developpers may choose to reject public keys
```
Comment (ernestognw) [inline contracts/utils/cryptography/RSA.sol:43]:
```suggestion
     * That is the default value many libraries use, such as OpenSSL. Developers may choose to reject public keys
```

File: contracts/utils/cryptography/RSA.sol [hunk 2]
```diff
@@ -36,8 +36,12 @@ library RSA {
...
      *
      * WARNING: This verification algorithm doesn't prevent replayability. If called multiple times with the same
-     * digest, public key and (valid signature), it will return true every time. Consider including an onchain nonce or
-     * unique identifier in the message to prevent replay attacks.
+     * digest, public key and (valid signature), it will return true every time. Consider including an onchain nonce
+     * or unique identifier in the message to prevent replay attacks.
+     *
+     * WARNING: This verification algorithm support any exponent. NIST recommendation are to use `65537` (or higher).
+     * That is the default value many libraries use, such as OpenSSL. Developers may choose to reject public keys
+     * using a low exponent out of security concerns.
```
Comment (Amxx) [inline contracts/utils/cryptography/RSA.sol:44]:
```suggestion
     * WARNING: This verification algorithm supports any exponent. NIST recommends using `65537` (or higher).
     * That is the default value many libraries use, such as OpenSSL. Developers may choose to reject public keys
     * using a low exponent out of security concerns.
```

**Review by cairoeth — APPROVED** [review]
lgtm

**Review by ernestognw — APPROVED** [review]

**Comment by Morhef** [comment]
.

**Pull request merged**